### PR TITLE
Removed unsupported configuration attributes

### PR DESCRIPTION
**Pull request by rpetrusha**
## Removed unsupported configuration attributes

@davidsh, I've just removed the unsupported attributes. Would you mind taking a closer look at the topic, though? In addition, the description for `usesystemdefault` says that it must be set to false to enable an adaptive proxy, but the "typical adaptive proxy" example sets it to true. Should it be set to false, or is the description inaccurate?

### CODE REVIEW

File: docs/framework/network-programming/proxy-configuration.md
Comment (mairaw) [inline docs/framework/network-programming/proxy-configuration.md]: nit: did you mean to have that pipe at the end?

File: docs/framework/network-programming/proxy-configuration.md
Comment (mairaw) [inline docs/framework/network-programming/proxy-configuration.md]: why are we talking about .NET 2.0 and 1.1 only here?

**Review by mairaw — APPROVED** [review]
From a text perspective, changes look good but left a few comments for you. But this needs a review from the product team before merging.

File: docs/framework/network-programming/proxy-configuration.md
Comment (davidsh) [reply docs/framework/network-programming/proxy-configuration.md]: >On .NET Framework 2.0 when this value is set to `true`, the Internet Explorer proxy settings are not overridden by other proxy settings in the configuration file. On .NET Framework 1.1, the Internet Explorer proxy settings can be overridden by other proxy settings in the configuration file.

I would delete the above sentences.

**Review by davidsh — CHANGES_REQUESTED** [review]
Left some comments

**Pull request merged**
